lib/models/messages/auxiliary/realtimecash.dart

```diff
@@ -6,12 +6,16 @@ class SimpleRealtimeCashDonationEventModel extends MessageModel {
   final String currency;
   final double value;
   final String hash;
+  final String donor;
```

Comment:
Make these fields nullable instead of having a default value. It will make translations a bit easier too.